packages/types/src/json.ts

```diff
@@ -52,11 +52,18 @@ export interface ImageJSON {
   public_url: string;
 }
 
+// move CountryIso types from clerkjs to this repo. Hence the any type in country
+// packages/clerk-js/src/ui/elements/PhoneInput/countryCodeData.ts
+export interface ResponseHeaderOptionsJSON {
+  responseHeaders: { country: any };
+}
+
 export interface EnvironmentJSON extends ClerkResourceJSON {
   auth_config: AuthConfigJSON;
   display_config: DisplayConfigJSON;
   user_settings: UserSettingsJSON;
   organization_settings: OrganizationSettingsJSON;
+  meta: ResponseHeaderOptionsJSON;
```

Comment:
This needs to be country, right?

Reply:
@nikosdouvlis we need this for because of this `data: EnvironmentJSON `

Reply:
I see..
not a fan of this as EnvironmentJSON does not need to know of `meta` but lets revise when w need to read headers for a second place

Reply:
An alternative solution is to use a `Clerk.setCountry` function similar to `Clerk.updateClient` that is specific to this functionality until we have another use case and need to read from FAPI headers. Having said that, I'd expect the Env JSON to be responsible for this only if the country was returned by FAPI as part of the environment payload.

@raptisj let's add a story for this week to revise this please.